geem/static/geem/js/geem_form.js

```diff
@@ -65,6 +65,7 @@ function OntologyForm(domId, resource, settings, callback) {
 			$('li[role="menuitem"][data-ontology-id="' + self.entityId + '"]').addClass('active')
 
 			var entity = get_form_specification_component(self.entityId)
+			top.form.components = entity.components
```

Comment:
So the callback param above is a name of a function that will be called after form spec entity is built and all form rendering is done.  In form_portal.js line 111 there is a call to OntologyForm() that contains the function ref. to portal_entity_form_callback, so you could add "top.form.components = form.components" there?

Reply:
Done. I moved the call to update the ontology grid there too, so the ontology grid code is less spread out.